wagtail/wagtailembeds/blocks.py

```diff
@@ -3,6 +3,57 @@
 from wagtail.wagtailembeds.format import embed_to_frontend_html
 
 
+class EmbedValue(object):
+    """
+    Native value of an EmbedBlock. Should, at minimum, have a 'url' property
+    and render as the embed HTML when rendered in a template.
+    NB We don't use a wagtailembeds.model.Embed object for this, because
+    we want to be able to do {{ value.url|embed:max_width=500 }} without
+    doing a redundant fetch of the embed at the default width.
+    """
+    def __init__(self, url):
+        self.url = url
+
+    def __str__(self):
```

Comment:
This probably needs the ``@python_2_unicode_compatible`` decorator https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.encoding.python_2_unicode_compatible